src/FakeItEasy/ArgumentConstraintManagerExtensions.cs

```diff
@@ -37,7 +38,7 @@ public static class ArgumentConstraintManagerExtensions
         {
             Guard.AgainstNull(manager);
 
-            return manager.Matches(x => x is null, x => x.Write("NULL"));
+            return manager.Matches(x => NullArgumentConstraint.Instance.IsValid(x), NullArgumentConstraint.Instance.ConstraintDescription);

```

Comment:
Why use a lambda instead of the method group here? (like you did for `IsNull`)

Reply:
Compiler hated the lambda.

Reply:
Sorry, I'm slow today. Could you elaborate?